src/nostr-profile-badge/nostr-profile-badge.ts

```
@@ -0,0 +1,329 @@
+  private userProfile: NDKUserProfile = {
+  name: '',
+  image: '',
+  nip05: '',
+  };
+
+  private theme: Theme = 'light';
+
+  private isLoading: boolean = true;
+  private isError: boolean = false;
+
+  private onClick: Function | null = null;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)